lib/users/users.yml

```diff
@@ -10,7 +10,18 @@ attributes:
     id: password
     name: Password
     description: Make it secure!
-  - type: text
-    id: level
-    name: Access Level
-    description: (make multi-select plugin w/selection of CTs for `content-creator`) CHOOSE one - superuser/admin/content-creator
+  - type: select
```

Comment:
This was meant to be the user config. This .yml file is being sent through the `content-types` module to create forms.
1. should I put this file somewhere else?
2. should I split out roles and pull them in to this file? Not sure how to do that with yaml, so I'd need to program for that (no biggie, just want to do it best)